module/Photo/config/module.config.php

```diff
@@ -145,6 +145,32 @@
                             ],
                         ],
                     ],
+                    'setprofilephoto' => [
+                        'type' => 'Segment',
+                        'options' => [
+                            'route' => '/setprofilephoto/:photo_id',
+                            'constraints' => [
+                                'photo_id' => '[0-9]+',
+                            ],
+                            'defaults' => [
+                                'controller' => 'Photo',
+                                'action' => 'setProfilePhoto',
+                            ],
+                        ],
+                    ],
+                    'removeprofilephoto' => [
+                        'type' => 'Segment',
+                        'options' => [
+                            'route' => '/removeprofilephoto/:photo_id',
```

Comment:
You can make photo_id an optional route param here for nicer routes. Then you don't need to set it to `0`.

Reply:
I believe by changing this to `'/removeprofilephoto[/:photo_id]'`, but not sure, since I haven't touched zend framework in a while.